common/components/WalletDecrypt/components/WalletButton.tsx

```diff
@@ -86,24 +86,47 @@ export class WalletButton extends React.PureComponent<Props> {
       >
         <div className="WalletButton-inner">
           <div className="WalletButton-title">
-            {icon && <img className="WalletButton-title-icon" src={icon} />}
+            {icon && <img className="WalletButton-title-icon" src={icon} alt="Logo" />}
             <span>{name}</span>
           </div>
 
-          {description && <div className="WalletButton-description">{description}</div>}
-          {example && <div className="WalletButton-example">{example}</div>}
+          {description && (
+            <div
+              className="WalletButton-description"
+              aria-label="description"
+              aria-hidden={!isSecure}
+            >
+              {description}
+            </div>
+          )}
+          {example && (
+            <div
+              className="WalletButton-example"
+              aria-label="example"
+              aria-hidden={!isSecure ? true : false}
```

Comment:
`!!isSecure`

Reply:
Also, why would this be hidden if it's secure?

Reply:
^ same as previous comment

Reply:
Example is only used by insecure wallets, and description only by secure. So I think you can make this `true` and just never aria-hidden the description. I think it's fair to assume that examples will always be not-useful, and descriptions will always be useful.